stackdriver_exporter.go

```diff
@@ -6,15 +6,15 @@ import (
 	"os"
 	"strings"
 
+	"github.com/PuerkitoBio/rehttp"
+	"github.com/frodenas/stackdriver_exporter/collectors"
```

Comment:
Minor point, but this line has moved unnecessarily.

I think the pattern is to split imports into groups: stdlib, other projects, other packages in this project.